scripts/build.mjs

```diff
@@ -70,7 +80,19 @@ class Builder {
     // environment variable `__COMPAT_LAYER=RunAsInvoker` to force the installer
     // to run as the existing user.
     const env = { ...process.env, __COMPAT_LAYER: 'RunAsInvoker' };
+    const releaseBuildIndex = args.indexOf('--release');
+    const fullBuildVersion = this.getDescribedTag();
+    let ptn;
```

Comment:
What is `ptn`? It appears to be some kind of regex we run against the `git describe` output… but it's unclear what the intention is.

Reply:
The intention is twofold:
1. Always remove the leading 'v', if present
2. Strip the trailing commit version out of releases. For releases we only need to expose MAJOR-MINOR-SUB

The story wasn't explicit about part 2, but seemed right. Also the build utilities want a version string that conforms to that (because of the code-signing part)

Reply:
Renamed the variable.